src/pages/NewChairConsole.js

```diff
@@ -151,6 +190,30 @@ const NewChairConsole = () => {
     }
   };
 
+  const createElection = (darId, index) => {
+    if (darId !== null) {
+      let copy;
+      Election.createDARElection(darId)
+        .then(() => {
+          Notifications.showSuccess({text: "Election successfully opened"});
+          copy = cloneDeep(filteredList);
+          copy[index].election = Object.assign( {}, copy[index].election, {status: "Open", finalAccessVote: false});
+          setFilteredList(copy);
```

Comment:
Another thing to note: filteredList is a subset of the total election list, where the subset can be, at most, as large as the electionList itself. This is important because whenever a user updates the search terms on the search bar, it uses the electionList as the initial master list and then cuts it down for each term. Once that's done, the trimmed list is assigned as the filteredList.

When you update the filteredList, you're updating a list that exists only with the current search terms. If the user changes the search terms, the filteredList is reset, and you lose the updates you've made.

Updating the filteredList is the right move to show immediate changes on screen, but you'll also need to make updates to the electionList for future search updates.

Reply:
good thinking! I'll add an update to electionList as well. 

> I've noticed another issue where the button/status doesn't update once you open an election, even with the code updates. I think this is a slightly more complex issue and will spend some time looking into it.
> 
> To reproduce it, just open an election on any non-open election with nothing in the search bar. The reason why this is interesting is because if you have search terms present, the row updates properly. My guess this problem has more to do with my templates and not the button itself, but I'll need some time to look into it. In the meantime, I left one more comment on the code that does need to be addressed (I left an explanation as to why).

I'm not seeing this error, is this what you mean
https://user-images.githubusercontent.com/77015255/107695604-d9c24a00-6c7e-11eb-9f93-9137d4b70f56.mov

Reply:
Ok, I've figured it out (can't believe I didn't realize it sooner). The issue occurs on any page that's not page 1, and it happens due to index. Index is derived from the list of visible rows on the table, which means that the value will occur within the range [0, tableSize]. If we're on page 2 or greater, the index needs to be processed as such. 
```(ex: currentPage = 2, tableSize = 10, index = 0 ===> index on filteredList would be 10, not 0)```

Reply:
Oh wow! Good thinking! So, I will add a check for the current page before we get the index